homeassistant/components/websocket_api/http.py

```diff
@@ -467,24 +465,26 @@ async def _async_websocket_command_phase(
         # Command phase
         while not wsock.closed:
             msg = await wsock.receive()
+            type_ = msg.type
+            data = msg.data
```

Comment:
Could we use `msg_` prefix instead of `_` suffix?
```suggestion
            msg_type = msg.type
            msg_data = msg.data
```

Reply:
Agree that this is a little cleaner 👍